### ISSUE TITLE
Trouble getting this working

### ISSUE THREAD

**Issue by jaexplorer** [nandorojo/shared-animations#14]
Hi i'm having some trouble getting this working. 

I have a colour picker component I want that changes the colour of the app.
I have a footer component I want to take that animation value and assign it as the background colour. 

However it only works on first render, is this library still maintained. I would love to connect?

**Comment by nandorojo** [comment]
hey it’s not really maintained tbh. at this point you’d be best off just passing an animated value around your app using React Context

**Comment by jaexplorer** [comment]
Yeah the only issue is that I need to update that value from a nested child. I think i'm finding either the value from the hook is not causing a rerender to the footer or i'm unable to update the value properly. For example, i'm calling newValue from inside useDerivedValue in reanimated 2.
